keps/sig-windows/2258-node-log-query/README.md

```diff
@@ -226,34 +226,33 @@ tests will not be possible for this feature.
 
 ##### e2e tests
 
-We will add a test that query the kubelet service logs on Windows and Linux nodes.
-On Windows node, the same kubelet service logs will queried by explicitly
-specifying the log file. In Linux the explicit log file query will be tested by
-querying a random file in present in /var/log.
+Tests have been added that query the kubelet service logs on Linux nodes and
+Microsoft-Windows-Security-SPP logs on Windows nodes with various options.
 
-On the Linux side tests will be added to [kubelet node](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/kubelet.go)
-e2e tests. For Windows a new set of tests will be added to the existing
-[e2e tests](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/windows).
+These tests are part of the [kubelet node](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/node/kubelet.go)
+e2e tests that are run as a daily periodic job:
+- https://testgrid.k8s.io/sig-windows-master-release#capz-master-windows-alpha-nodelogquery
 
-- node: https://storage.googleapis.com/k8s-triage/index.html?sig=node
-- windows: https://storage.googleapis.com/k8s-triage/index.html?sig=windows
+This job runs tests against both Windows and Linux nodes.
 
 ### Graduation Criteria
 
 The plan is to introduce the feature as alpha in the v1.27 time frame behind the
-`NodeLogQuery` kubelet feature gate and using the `kubectl alpha node-logs`
-sub-command.
+`NodeLogQuery` kubelet feature gate and `enableSystemLogQuery` kubelet option.
 
 #### Alpha -> Beta Graduation
 
-The plan is to graduate the feature to beta in the v1.29 time frame. At that
-point we would have collected feedback from cluster administrators and
-developers who have enabled the feature. In addition we will provide a kubectl
-plugin for querying the logs more elegantly instead of using raw API calls.
+The plan is to graduate the feature to beta in the v1.30 time frame. So far we
+have not received any negative feedback from cluster administrators and
+developers who have enabled the feature.
+
+A [kubectl plugin](https://github.com/aravindhp/kubectl-node-logs) has been released
+and added to the Krew [index](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/node-logs.yaml)
+for querying the logs more elegantly instead of using raw API calls.
```

Comment:
SIG-cli was involved in the design and the decision was to not make this a core command at any point. This was also discussed during a SIG-arch call to drive the feature to acceptance. PS: this will always be a feature that defaults to off.

Reply:
> Have we discussed with SIG CLI if this might become a core command later?

Yes it was discussed, but given this functionality is off by default, and there are some security concerns wrt that, we've decided it's best to keep it optional and the same applies to kubectl plugin.

Reply:
Ack, thanks!